src/opt.rs

```diff
@@ -0,0 +1,33 @@
+use core::fmt::{Display, Formatter};
+
+use crate::Argument;
+
+/// A short or long option.
+#[derive(Copy, Clone, Debug, Hash, PartialEq, Eq)]
+pub enum Opt<'a, A: Argument + 'a> {
```

Comment:
What is Argument? Did you accidentally commit half of your "generics" work?

Reply:
oops, where did that come from? Let me fix that...